### DISCUSSION TITLE
Display a breakdown of the user's PP in the results screen

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#15566] [Ideas] [answered]
Show breakdown of pp (aim, speed, accuracy, etc.) in the result screen.

**Comment by bdach** [comment]
https://github.com/ppy/osu/issues/9463

**Reply by smoogipoo** [reply]
This isn't the same, it's about showing PP breakdown (and, as I would like, compared to a perfect play) rather than star rating breakdown.

**Reply by developomp** [reply]
I've read that issue. This is completely different.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15570
